src/stomach.cpp

```diff
@@ -6,12 +6,14 @@
 
 #include "cata_utility.h"
 #include "character.h"
-#include "game.h"
 #include "json.h"
 #include "stomach.h"
 #include "units.h"
 #include "vitamin.h"
 
+// NOLINTNEXTLINE(cata-static-declarations)
+extern const int savegame_loading_version;
```

Comment:
VS is choking on that for some reason. I will take a look, but do we even need code that uses it anymore? I guess 0.F-1 characters should do fine without it.